var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -29,7 +29,6 @@ class Mfem(Package):
     """Free, lightweight, scalable C++ library for finite element methods."""
 
     homepage = 'http://www.mfem.org'
-    url      = 'https://github.com/mfem/mfem'
```

Comment:
If you can find real URLs, yes! I tried once but couldn't find them. I didn't look very hard.